src/main/java/adapters/default_game/GamePanelPresenter.java

```diff
@@ -17,7 +17,7 @@ public class GamePanelPresenter extends JPanel implements IGamePanelOutputBounda
     final int PANEL_HEIGHT = MazeInfo.getPanelHeight();
     private int playerX;
     private int playerY;
-    private int playerStamina;
+    public int playerStamina;
```

Comment:
Is there any reason why you changed it to public? No other classes seem to be using this variable.

Reply:
It is changed back to private now :)